Source/Core/UICommon/GameFileCache.cpp

```diff
@@ -111,8 +111,8 @@ bool GameFileCache::Update(
         cache_changed = true;
         --end;
         *it = std::move(*end);
-        m_cached_files.pop_back();
       }
+      m_cached_files.erase(it, m_cached_files.end());
```

Comment:
Are you sure this works correctly? It was one line further down originally.

Reply:
Oops. It's definitely incorrect.